bin/kano-settings-onboot

```diff
@@ -429,13 +472,22 @@ if __name__ == '__main__':
         set_to_HDMI(False)
 
     # calculate changes needed
-    changes = compare_and_set_mode() or \
-        compare_and_set_full_range() or \
-        compare_and_set_overscan() or \
-        compare_and_set_optimal_resolution()
+    overall_changes = False
 
-    if changes:
+    changes = compare_and_set_mode(edid, status)
+    overall_changes = overall_changes or changes
+
+    changes = compare_and_set_full_range(edid, status)
+    overall_changes = overall_changes or changes
 
+    changes = compare_and_set_overscan(edid, status)
+    overall_changes = overall_changes or changes
+
+    changes = set_resolution_for_screen_kit(supported) or \
+              compare_and_set_optimal_resolution(edid, status, supported, verbose)
+    overall_changes = overall_changes or changes
+
+    if changes:
```

Comment:
Thanks for the changes Radu. This is the only line that I'm still concerned about - should this `changes` only be the last version of the values that you check or should it be `overall_changes`? Obviously you keep overwriting `changes` and I just want to check that this is what you intend.